Tests/Marketplace/Tests/search_and_install_packs_test.py

```diff
@@ -245,3 +246,25 @@ def test_not_find_malformed_pack_id():
     with pytest.raises(Exception, match='The request to install packs has failed. '
                                         'Reason: This is an error message without pack ID'):
         script.find_malformed_pack_id('This is an error message without pack ID')
+
+@timeout_decorator.timeout(3)
```

Comment:
why is this needed?

Reply:
Because theoretically this can lead to an endless loop